code/index.html

```diff
@@ -4,27 +4,140 @@
+      </video>
+     
+    </section>
 
-    <!-- Signup form -->
 
-    <section class="accordion">
       <!-- Accordion -->
-      <h1>FAQ</h1>
 
-      <div class="question" id="section1">Section 1</div>
+
+    <section id="accordion">
+
+      <h3>FAQ</h3>
+      <div class="questions-box">
+      <article>
+        <div class="question" id="section1">
+          <button type="button">What songs will we learn?</button></div>
+        <div class="answer">
+          <p>All the classics! You'll learn everything from Wonderwall and House of the Rising Sun, to Shape Of You and Fast Car. Become the life of the party at every outdoor gathering, such as campsites and barbecues!
+          </p>
+        </div>
+      </article>
+
+      <article>
+      <div class="question" id="section2">
+        <button type="button">What's the tuition fee?</button></div>
+      <div class="answer">
+        <p>Our pricing starts at 20€ per week with the possibility of private lessons for an additional fee. Please fill out your requests and questions in the submission form.</p>
+      </div>
+      </article>
+
+
+      <article>
+      <div class="question" id="section3">
+        <button type="button">Is this guaranteed to make me cooler?</button></div>
       <div class="answer">
-        <p>Lorem ipsum...</p>
+        <p> Being able to play guitar is proven to make you appear 25% cooler than non-guitar-playing individuals. However, it's important to take into account ones base-level of initial coolness.
+        </p>
       </div>
+      </article>
 
-      <div class="question" id="section2">Section 2</div>
+      <article>
+      <div class="question" id="section4">
+        <button type="button">The people I live with don't want me to take this class?</button></div>
       <div class="answer">
-        <p>Lorem ipsum...</p>
+        <p>Disgruntled roomates and family members are, unfortunately, a common problem when practising the guitar. Try offering them snacks and earplugs! </p>
       </div>
+      </article>
+    </div>
+
     </section>
+
+
+    <!-- Signup form -->
+
+    <section id="form-section">
+
+          <div class="form-box">
+            <div class="signup"><h3>Sign up today!</h3></div>
+            
+              
+            <div class="left">  
+              <form name="myForm" action="http://httpbin.org/anything" method="post">
+              
+                
+              <label for="fname">Name:</label><br>
+                <input type="text" id="fname" name="firstname" required><br>
+              <label for="email">Email:</label><br>
+                <input type="email" id="email" name="emailname" required><br>
+              <label for="nmbr">Phone:</label><br>
+                <input type="number" id="nmbr" name="phonenumber"><br> 
+              <label for="message">Custom message<br> </label><br>
+                  <textarea name="message" rows="5" cols="50%">Special requests etc</textarea><br>
```

Comment:
what does the <br> do in the form? I can’t see in the inspector what their function is there

Reply:
I used br to break the line between the label texts and the form boxes!